.github/ISSUE_TEMPLATE/discussions.yaml

```diff
@@ -0,0 +1,20 @@
+name: "💬Discussions, 💭Ideas or❓Questions"
+description: "Discussion/Brainstorming/Questions for LDAR-Sim"
+title: "[MISC] Brief Description"
+labels: ["discussion"]
```

Comment:
Does a discussion label exist in the LDAR-Sim labels? If not can you create one when this gets merged.

Reply:
Yes, the discussion tag got created during the process of getting this template ready.